src/arch/xen/include/COPYING

```diff
@@ -0,0 +1,23 @@
+Source files taken from the Xen distribution are governed by their original license.
```

Comment:
This is unusual file or place for it

Reply:
It belong to Xen port at least. I have no strong opinion where would it belong. `src/arch/xen` (one level upper)?

Reply:
Yes, one level upper looks better.
But it means than all code in xen folder is under this license?

Reply:
License specifies xenstore related files, it's xensotre.c and xenstore.h